docs/topics/db/queries.txt

```diff
@@ -1088,7 +1088,7 @@ For example, a ``Blog`` object ``b`` has access to a list of all related
 All examples in this section use the sample ``Blog``, ``Author`` and ``Entry``
 models defined at the top of this page.
 
-.. _descriptors: http://users.rcn.com/python/download/Descriptor.htm
+.. _descriptors: https://docs.python.org/3/howto/descriptor.html
```

Comment:
I'd rather use intersphinx to the Python docs but I guess it's not possible now because you can't use `:doc:` as far as I know, and there aren't any refs to link to in the Python doc.